manifests/x/Xiaoe/Xiaoetong/1.10.13/Xiaoe.Xiaoetong.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Xiaoe.Xiaoetong
+PackageVersion: 1.10.13
+InstallerType: nullsoft # electron-builder
+Scope: machine
+InstallerSwitches:
+  Upgrade: --updated
+UpgradeBehavior: install
+Protocols:
+- xiaoe-ilc
+ReleaseDate: 2022-11-25
```

Comment:
```suggestion
ReleaseDate: 2022-11-25
AppsAndFeaturesEntries:
- Publisher:
```
Publisher ARP entry is null
<img src=https://user-images.githubusercontent.com/88161975/205152867-23b5ea0b-a796-4b42-a41a-234ab56ca0cd.png height=70>

Reply:
Technically the package will still work without this, since the name and display version match